packages/sanity/src/desk/components/breadcrumbs/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,55 @@
+import {Text, Tooltip, Box, type ButtonProps} from '@sanity/ui'
+import React, {forwardRef, type HTMLProps, type ReactNode, type ForwardedRef, useRef} from 'react'
+import {BreadcrumbButtonRoot, BreadcrumbItemSpan} from './breadcrumbs.styles'
+
+type ParentTextProps = ButtonProps & Omit<HTMLProps<HTMLButtonElement>, 'ref' | 'as'>
+
+/**
+ * @internal
+ * @hidden
+ */
+export interface BreadcrumbItemProps extends ParentTextProps {
+  children: ReactNode
+  isTitle?: boolean
+}
+
+/**
+ * @internal
+ * @hidden
+ */
+export const BreadcrumbItem = forwardRef(function BreadcrumbItem(
+  props: BreadcrumbItemProps,
+  ref: ForwardedRef<HTMLButtonElement>
+) {
+  const {children, isTitle, ...restProps} = props
+  const textRef = useRef<HTMLSpanElement | null>(null)
+  const {clientWidth = 0, scrollWidth = 0} = textRef.current ?? {}
+
+  return (
+    <BreadcrumbButtonRoot
+      isTitle={isTitle}
+      ref={ref}
+      mode="bleed"
+      padding={2}
+      textAlign="left"
+      justify="flex-start"
+      {...restProps}
+    >
+      <Tooltip
+        content={
+          <Box padding={2} style={{maxWidth: '500px'}}>
+            <Text size={1}>{children}</Text>
+          </Box>
+        }
+        padding={1}
+        placement="bottom"
+        fallbackPlacements={['top', 'left', 'right']}
+        portal
+        // We only want to show tooltip if the text is overflowing
+        disabled={clientWidth >= scrollWidth}
```

Comment:
Two things here that I am assuming is causing this issue. 

1. The breadcrumb is supposed to dynamically resize based on space. The title has min width of 220px and if it's getting smaller than that then the rest of the items are supposed to go inside a popover which is not yet implemented yet which is causing some weird styling issues
2. The tooltip is only supposed to show when the items have ellipsis truncating happening. This logic might be a little brittle and need some more testing after the first bullet is implemented since the truncating happens with CSS and we are trying to figure out if it's truncating using JS which may or may not have performance issues.

Thanks for pointing this out, I have noted these requirements in the ticket for it